### DISCUSSION TITLE
Jump from Debian to Spiral?

### DISCUSSION THREAD

**Discussion by Tagby76** [SpiralLinux/SpiralLinux-project#235] [Q&A] [answered]
Is there a way I can install vanilla Debian and then switch repositorys and install Spiral's configurations automagically?

I know you can do this with Endeavor OS to Arch Linux. Or was vice versa?

Anyway I want to know if it's possible to do this with Spiral.

**Comment by geckolinux** [comment]
Hi there,

It's possible, you can find all of the configurations that make SpiralLinux unique in the tar.gz archives here on GitHub.

But the (respectful) question is _why_? Unless you already have a working vanilla Debian installation I can't think of any reason to go through that manual process. It would be easier and faster to simply backup the `/home/` directories and install SpiralLinux and restore the `/home/` data.

**Comment by Tagby76** [comment]
You really wanna know?  Omg, it's so dumb.  I'm afraid of AppArmor.  That's it!  That's the main reason!  Debian said they've got AppArmor working out of the box, no configuration required, and I really want/need/mostly want AppArmor.  I like the warm n' fuzzies it gives me knowing it's on my system and fully setup.  I've never installed AppArmor.  I think I tried once on EndeavourOS/Arch Linux, quit b/c too much reading (it was so daunting!), and gave up.

I'm looking for a distro that will give me hardened SELinux/AppArmor profiles OOTB, a grsecurity kernel, and Btrfs with full-disk encryption.

That's what I'm really after.  Can you help me?

**Reply by geckolinux** [reply]
I see. That should be easy to install on top of SpiralLinux:
https://wiki.debian.org/AppArmor/HowToUse#Install_AppArmor
Just install the required packages, and then you can use GRUB Customizer (installed by default) to add the GRUB `apparmor=1 security=apparmor` boot parameter.

**Reply by Tagby76** [reply]
Hey thanks for the link. I really appreciate that. I'll definitely install that.

**Comment by Tagby76** [comment, marked as answer]
>I see. That should be easy to install on top of SpiralLinux:
https://wiki.debian.org/AppArmor/HowToUse#Install_AppArmor
Just install the required packages, and then you can use GRUB Customizer (installed by default) to add the GRUB apparmor=1 security=apparmor boot parameter.

via @geckolinux  (https://github.com/SpiralLinux/SpiralLinux-project/discussions/235#discussioncomment-5266866)
